src/js/login/containers/Main.jsx

```diff
@@ -75,6 +80,7 @@ class Main extends React.Component {
       });
       const receiver = window.open(`${myLoginUrl}&op=signin`, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
       receiver.focus();
+      this.getLoginUrl();
```

Comment:
Mmm...maybe the following are questions for @tiffanybbrown-va:

So if I'm reading this right, `componentDidMount` fetches the login url from...wherever `environment.API_URL` points to (vets-api? ID.me's api?). It passes that to `this.props.onUpdateLoginUrl()` which (I assume) updates `this.props.loginUrl.first`. We then use that to log in via `handleLogin()`.

Line 76 reads:
```js
const myLoginUrl = this.props.login.loginUrl.first;
```
That's the line that gets the url to actually log in on line...81? After we open that window, we go fetch another url.

Knowing as little as I do about this, I'm wondering why fetching a new url afterward would be necessary. I mean, doesn't it get a new url every time the component mounts? So are all these resubmissions people who login, do their thing, log out, but never leave the site, refresh, close the tab / window, or anything like that?

I'm probably missing something, but this seems like it'll fetch a login url before login and after login, but the second one probably won't be used most of the time. Would it make more sense to fetch the url at the beginning of `handleLogin()`, or is that in `componentDidMount` because it was taking a while and optimized because of that? Or was it because we need it in the redux store (I assume that's where the prop with the url comes from) before we log in for some reason?

Sorry for being so vague, but I'm trying to wrap my head around the whole problem before I can say this is "the right" solution.